Jenkinsfile

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env groovy
-
-library("govuk")
-
-node {
-  govuk.buildProject()
-}
```

Comment:
Does anything else need to be done to remove this project from Jenkins?

Reply:
It's possible that it's referenced in alphagov/govuk-puppet.